wordfreq/transliterate.py

```diff
@@ -0,0 +1,70 @@
+# This table comes from https://github.com/opendatakosovo/cyrillic-transliteration/blob/master/cyrtranslit/mapping.py,
+# from the 'cyrtranslit' module, which is too flaky for us to just import.
```

Comment:
Sure, I'll get to the point: the cyrtranslit package is supposed to be compatible with Python 3, and its examples look like Python 3, but it uses import syntax that's Python 2 only. I've got a bug report open about that.

Using our own version of the table lets us handle mixed languages better, anyway.